src/components/staking/staking-form.tsx

```diff
@@ -224,18 +262,7 @@ export default function StakingForm({
                 </div>
                 <FormControl>
                   <div className="relative">
-                    <Input
-                      placeholder={`eg: ${
```

Comment:
I guess you removed the display of the `maxTokenAllowed` in the placeholder because it now needs to take into account the available token amount of the pool, which is fine, but the same issue also applies to the "max" button bellow and to the zod validations of the form.
Maybe you can just update the definition of `maxTokenAllowed` line 60 to take into account the available token amount of the pool and put back the placeholder.

Reply:
I removed the placeholder because we had a lot of side effects showing an infinite-like number. I added the test if it was not maxUint, but if you deposit even one token, you're not in maxUint anymore, so the display was really bad.
This is just much simpler, even less accurate. If the placeholders don't match the client, they can customize it easily.